tests/conftest.py

```diff
@@ -1415,6 +1439,38 @@ def titanic_data_context_stats_enabled_config_version_3(tmp_path_factory, monkey
     return ge.data_context.DataContext(context_path)
 
 
+@pytest.fixture(scope="module")
+def titanic_spark_db(tmp_path_factory, spark_warehouse_session):
+    titanic_database_name: str = "db_test"
+    titanic_table_name: str = "tb_titanic"
+
+    titanic_csv_path = file_relative_path(__file__, "./test_sets/Titanic.csv")
```

Comment:
@lccasagrande Type hints, please. :-).   Thank you!

Reply:
Fixed